cli/tests_setup.sh

```diff
@@ -14,40 +14,54 @@ SERVER_URL="${SERVER_URL:-https://localhost:8000}"
 DIRECTORY="${DIRECTORY:-/tmp/medperf_test_files}"
 CLEANUP="${CLEANUP:-false}"
 FRESH="${FRESH:-false}"
-MEDPERF_STORAGE=~/.medperf
-MEDPERF_LOG_STORAGE=~/.medperf_logs
+
+TEST_ROOT="/tmp/medperf_tests_$(date +%Y%m%d%H%M%S)"
+export MEDPERF_CONFIG_PATH="$TEST_ROOT/config.yaml"  # env var
+MEDPERF_STORAGE="$TEST_ROOT/storage"
+SNAPSHOTS_FOLDER=$TEST_ROOT/snapshots
+
+MEDPERF_LOG_PATH=~/.medperf_logs/medperf.log
 SERVER_STORAGE_ID="$(echo $SERVER_URL | cut -d '/' -f 3 | sed -e 's/[.:]/_/g')"
 TIMEOUT="${TIMEOUT:-30}"
 VERSION_PREFIX="/api/v0"
 LOGIN_SCRIPT="$(dirname $(realpath "$0"))/auto_login.sh"
 ADMIN_LOGIN_SCRIPT="$(dirname $(dirname $(realpath "$0")))/server/auth_provider_token.py"
 MOCK_TOKENS_FILE="$(dirname $(dirname $(realpath "$0")))/mock_tokens/tokens.json"
-
+SQLITE3_FILE="$(dirname $(dirname $(realpath "$0")))/server/db.sqlite3"
 echo "Server URL: $SERVER_URL"
-echo "Storage location: $MEDPERF_SUBSTORAGE"
 
+ev() {
+    local timestamp=$(date +%m%d%H%M%S)
+    local formatted_cmd=$(echo "$@" | sed 's/[^a-zA-Z0-9]\+/_/g' | cut -c 1-50)
+    LAST_SNAPSHOT_PATH="$SNAPSHOTS_FOLDER/${timestamp}_${formatted_cmd}.sqlite3"
```

Comment:
Could we end up clutteraing this folder indefinitely if we're not cleaning it consistently? Would there be a way of defining a max amount of snapshots that can exist at a given time?

Reply:
Not exactly
```
tmp/
|-- medperf_tests_20240409140527/
|--|-- snapshots/ # here all the snapshots for that run are stored
|--|--|--....
|--|-- storage/
|--|--|-- .medperf/
|--|--|--|-- ....
|--|-- config.yaml
```

So, this snapshots folder belongs to a specific tests run, not mixing with snapshots from other runs. No need to manage especially snapshots. If you don't need this test results anymore (or you use `-c`/`CLEAN` flag), you delete a whole `medperf_tests_XXXYYY`.

However by default `-c` is set up to `False`. So, we are cluttering `/tmp/` folder with a bunch of `medperf_tests_XXXYYY`.

What do you think guys if we ultimately remove test root folder if tests were passed successfully? And oldish crap after failed tests we can remove manually.

Or I can implement a